playwright/tests/changes/changes.spec.ts

```diff
@@ -1,5 +1,134 @@
+import { expect } from "@playwright/test";
+
 import { test } from "../../fixtures";
+import { mockManagerProcedures } from "../../utils";
+import { emptyLibraries, simpleCustomType } from "../../mocks";
+
+test.describe("Changes", () => {
+  test.run({ loggedIn: true })(
+    "I cannot see the login screen when logged in",
+    async ({ changesPage }) => {
+      await changesPage.goto();
+      await expect(changesPage.loginButton).not.toBeVisible();
+      await expect(changesPage.notLoggedInTitle).not.toBeVisible();
+      await expect(changesPage.notAuthorizedTitle).not.toBeVisible();
+    },
+  );
+
+  test.run()(
+    "I can see the login screen when logged out",
+    async ({ changesPage }) => {
+      await changesPage.goto();
+      await expect(changesPage.loginButton).toBeVisible();
+      await expect(changesPage.notLoggedInTitle).toBeVisible();
+    },
+  );
```

Comment:
The fact that this tests runs when logged out is not obvious. Should we make it obvious with something like `test.run({ loggedIn: false })`?

(I suggested making `loggedIn` into a helper function (`await logIn()`) in a previous comment. In this case, we would probably also make a `await logOut()` helper to make it obvious.)

Reply:
For this one I agree that we could call it like this:

```
test.run({ loggedIn: false })(
    "I can see the login screen when logged out",
    async ({ changesPage }) => {
      await changesPage.goto();
      await expect(changesPage.loginButton).toBeVisible();
      await expect(changesPage.notLoggedInTitle).toBeVisible();
    },
  );
  ```
  
  But for the other one not related to login I'm not sure to force dev to write down every test `{ loggedIn: false }`. Because if it's not necessary to be logged in, better to test without a login. It's quicker and also ensure it's not mandatory. It can be too much to always have to setup the test while 85% of the tests need the same default, no? WDYT?